p8e-api/src/main/kotlin/io/provenance/engine/service/ChaincodeInvokeService.kt

```diff
@@ -209,7 +209,9 @@ class ChaincodeInvokeService(
                 val txBody = batch.map {
                     it.attempts++
                     it.request
-                }.toTxBody()
+                }.toTxBody().toBuilder()
+                    .setTimeoutHeight(currentBlockHeight + chaincodeProperties.blockHeightTimeoutInterval)
```

Comment:
If https://github.com/provenance-io/p8e/pull/86/files#diff-8c67dfe07e7217aa7039fcc68ae6edb2923d579c1f1d3c63a3daacf49a786664R180 is strung out to the max in the perfect scenario, could currentBlockHeight be at most 5 blocks behind? Not that it's a big deal, but might want to account for that and slightly increase whatever we use for `blockHeightTimeoutInterval`?

Reply:
@scirner22 I'm not quite sure what you are referring to with that link

Reply:
It seems like the aim was to have the timeout such that the transaction has 1 minute worth of blocks. I think our notion of currentBlockHeight might be at most 5 blocks behind what the current height really is. If that seems correct, maybe we just want the interval to be 20 blocks or something?

Reply:
ok, that seems reasonable. I think the current block height should be fairly close usually since we grab it at each loop iteration, but if the batching logic takes a while in some case we could probably fall behind a bit. I'll change the default to 20